router/notesRouter.js

```diff
@@ -0,0 +1,81 @@
+// Router Model to keep things easy and readable
+const express = require("express")
+const router = express.Router();
+const notes = require('../helpers/notesModel')
+
+// To get all the notes
+router.get('/get/all', (req, res) => {
+
+ notes.fetchAll()
+     .then(notes => {
+         res.status(200).json(notes)
+        })
+        .catch(err =>{
+            res.status(500).send(err)
+        })
+
+})
+
+//To get a single note by id
+
+router.get('/:id', async (req, res) => {
+    const { id } = req.params;
+    await notes.fetchNote(id)
+        .then(note => {
+            note
+                ? res.status(200).json(note)
+                : res.status(400).json({error: "The note does not exist"})
+        })
+        .catch(err => {
+            res.status(500).send(err)
+        })
+})
+
```

Comment:
Endpoints look great Stefan 👍 , as an exercise and practice for building more complex backends you could try your hand at adding some user-input validation middleware.